### ISSUE TITLE
[Missing]: Wolf armor not appearing on dogs

### ISSUE THREAD

**Issue by Nov1st2k10** [ViaVersionAddons/ViaBackwards-Plus#8]
### What items are missing?

 Wolf armor not appearing on dogs

### Checks

- [X] I downloaded the latest version of the repository
- [x] The server has the latest snapshot of the Via* plugins

### Which one of the detector mods are present?

Chime

### Your Minecraft version

1.20.1

**Comment by bangetto** [comment]
yeah, but if Via doesn't sends over the dog variant info then probably it doesn't send what the dog is wearing so, I can't fix this

**Comment by bangetto** [comment]
closing as the issue moving forward is obsolete
